student.rb

```diff
@@ -1,6 +1,8 @@
-require relative 'person'
+require './person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
```

Comment:
- [ ] You would need a method that serves as the 'belongs-to' side (a student belongs to a classroom). It should be something like this 
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```
You can find more information about it [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md)